R/set-defaults.R

```diff
@@ -0,0 +1,166 @@
+# ENVIRONMENT ----
+# Environment that holds various global variables and settings for headliner,
+# such as the current default headline syntax. It is not exported.
+headliner_global <- new.env(parent = emptyenv())
+
+# Initial default values
+# The list of articles to be used for 'get_articles()'
+headliner_global$articles <- list(addl_a = "", addl_an = "")
+
+default_headline <- "{trend} of {delta} ({orig_values})"
+default_orig_values <- "{c} vs. {r}"
+
+headliner_global$headline <- default_headline
+headliner_global$orig_values <- default_orig_values
+
+
+# VIEW ----
+#' List the defaults that headliner is using
+#'
+#' The default phrases used by glue might not be easy to remember. This
+#' function will show the values headliner uses as default.
+#' @export
+#' @seealso [augment_article_patterns()], [set_headliner_defaults()]
+#'
+#' @examples
+#' show_headliner_defaults()
+show_headliner_defaults <- function() {
```

Comment:
I think this name definitely works, I just wonder if you might get more out of something more specific like `show_headliner_phrasing` or `show_headliner_syntax`. Same goes for `set_headliner_defaults` obviously

Reply:
Eventually I plan to give it more, like default # of digits or the multiplier to use if you know you're mostly going to be giving it fractions.